.github/spawn-dailies.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 latest_master_url="https://download.nextcloud.com/server/daily/latest-master.tar.bz2"
-latest_stable22_url="https://download.nextcloud.com/server/daily/latest-stable22.tar.bz2"
+latest_stable23_url="https://download.nextcloud.com/server/daily/latest-stable23.tar.bz2"
+latest_stable24_url="https://download.nextcloud.com/server/daily/latest-stable24.tar.bz2"
```

Comment:
Agh, I've been asking for this for months, and they finally put it in place without telling me :man_facepalming:. Thank you!